### PR TITLE
Bani-2022-12-07

### PR DESCRIPTION
**Pull request by Bani2791**
## **1. Tidyup: Customer > Stats**

https://bani-2022-12-07--connexcs-docs.netlify.app/customer/stats/

## **2. Tidyup: Customer > Route Stats**

https://bani-2022-12-07--connexcs-docs.netlify.app/customer/route-stats/

## **3. Tidyup: Customer > Route Stats**

https://bani-2022-12-07--connexcs-docs.netlify.app/customer/latest-calls/

## **4. Tidy up: Customer > Dialogs**

https://bani-2022-12-07--connexcs-docs.netlify.app/customer/dialogs/

# **5. Tidy up: Customer > Alert**

https://bani-2022-12-07--connexcs-docs.netlify.app/customer/alerts/

# **6. Tidy up: Customer > DID**

https://bani-2022-12-07--connexcs-docs.netlify.app/customer/did/

### CODE REVIEW

File: docs/customer/dialogs.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
+**Dialogs** display the status of current active calls on the account.
 
-!!! note "Ended Status"
-    "Ended" indicates a call is over, but the telephony switches are still tearing down the call, including the Real-time Transport Protocol (RTP) stream and any final signaling. After that, the server gathers data for billing. 
+Use the **Refresh** button to get up-to-date information.
 
+!!! note "Ended Status"
+    "Ended" indicates a call is over, but the telephony switches are still tearing down the call, including the Real-time Transport Protocol (RTP) stream and any final signaling. After that, the server gathers data for billing.
 !!! note "Global Dialogs"
     You can also configure Dialogs for specific Customers in **Global :material-menu-right: Dialogs**, which displays all configured Dialogs.
```
Comment (syedrizwansrs) [inline docs/customer/dialogs.md:12]: If you can rewrite this please, basically this option lets a customer see all of their customer's active call.

**Review by syedrizwansrs — APPROVED** [review]
Looks ok.

**Review by syedrizwansrs — APPROVED** [review]

**Review by syedrizwansrs — APPROVED** [review]

**Review by syedrizwansrs — APPROVED** [review]

**Review by syedrizwansrs — APPROVED** [review]

**Review by syedrizwansrs — APPROVED** [review]

**Review by syedrizwansrs — APPROVED** [review]

**Review by syedrizwansrs — APPROVED** [review]

**Review by syedrizwansrs — APPROVED** [review]

**Review by syedrizwansrs — APPROVED** [review]

**Review by syedrizwansrs — APPROVED** [review]

**Review by syedrizwansrs — APPROVED** [review]

**Review by syedrizwansrs — CHANGES_REQUESTED** [review]
If you can rewrite this please, basically this option lets a customer see all of their customer's active call.

File: docs/customer/dialogs.md (hunk 1 above)
Comment (Bani2791) [reply docs/customer/dialogs.md:12]: ## Re-written the Global Dialogs

note "Global Dialogs"
    You can also see the status of all the calls for specific Customers in **Global :material-menu-right: Dialogs**.

    Also, you can select and end the calls using **End Calls** and the calls are terminaed using the MI Termination.

**Review by syedrizwansrs — APPROVED** [review]

**Comment by syedrizwansrs** [comment]
Approved

**Pull request merged**